cli/cmd/publish.go

```diff
@@ -69,13 +68,16 @@ type PackageJSONVersion struct {
 	SchemaVersion int `json:"schema_version"`
 }
 
+type Kind = cloudquery_api.PluginKind
+
 type PackageJSONV1 struct {
-	Name             string             `json:"name"`
-	Message          string             `json:"message"`
-	Version          string             `json:"version"`
-	Protocols        []int              `json:"protocols"`
-	SupportedTargets []TargetBuild      `json:"supported_targets"`
-	PackageType      plugin.PackageType `json:"package_type"`
+	Name             string                    `json:"name"`
+	Message          string                    `json:"message"`
+	Version          string                    `json:"version"`
+	Kind             cloudquery_api.PluginKind `json:"kind"`
```

Comment:
I could also make it a string or any other wrapper type and convert, but this works as we keep the enums consistent